packages/e2e-tests/plugins/iframed-multiple-stylesheets/block.json

```diff
@@ -0,0 +1,15 @@
+{
+	"apiVersion": 2,
+	"name": "test/iframed-multiple-stylesheets",
+	"title": "Iframed Multiple Stylesheets",
+	"category": "text",
+	"icon": "smiley",
+	"description": "",
+	"supports": {
+		"html": false
+	},
+	"textdomain": "iframed-multiple-stylesheets",
+	"editorScript": "iframed-multiple-stylesheets-editor-script",
+	"editorStyle": "file:./editor.css",
+	"style": ["iframed-multiple-stylesheets-style", "iframed-multiple-stylesheets-style2"]
```

Comment:
You can run `npm run format-js packages/e2e-tests/plugins/iframed-multiple-stylesheets/block.json` to format this file with Prettier.

Reply:
There are no spaces around the array item. You can ignore that though, we need to address that on the project's level 😄 

```json
"style": [ "iframed-multiple-stylesheets-style", "iframed-multiple-stylesheets-style2" ]
```